pkg/cloudprovider/aws/instance.go

```diff
@@ -103,7 +103,7 @@ func (p *InstanceProvider) Create(ctx context.Context,
 		return nil, fmt.Errorf("expected 1 instance ids, but got %d due to errors %v", count, createFleetOutput.Errors)
 	}
 	if count := len(createFleetOutput.Errors); count > 0 {
-		zap.S().Warnf("CreateFleet encountered %d errors, but still launched instances, %v", count, createFleetOutput.Errors)
+		zap.S().Debugf("CreateFleet encountered %d errors, but still launched instances, %v", count, createFleetOutput.Errors)
```

Comment:
Can we minimize the output here? It's super spammy. Maybe just the error codes?

Reply:
I'm concerned that minimizing too much will cause clutter without any useful information.  The whole Errors response is quite confusing in general. It might actually be safe to check if the desired number of instances were launched and only print errors if there is a mismatch. That would reduce the logs here to only situations where the resulting errors had a meaningful impact. If we only show error codes, we might just see a bunch of 
"InsufficientCapacity", which doesn't tell you how to fix anything. Wdyt about starting with reducing and if we still find them noisy we can do some more firm reduction?

Reply:
Sounds good!